autohoot/autodiff.py

```diff
@@ -838,6 +838,14 @@ def set_inputs(self, nodes):
         self.shape = self._output_shape(self.einsum_subscripts, nodes)
 
     def compute(self, input_vals):
+        # Ensure einsum_subscripts are valid in backends
+        einsum_symbols = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ.,->'
```

Comment:
Use this?
https://github.com/tensorflow/tensorflow/blob/590d6eef7e91a6a7392c8ffffb7b58f2e0c8bc6b/tensorflow/python/ops/special_math_ops.py#L330

Reply:
done